client/clientmanager/src/main/java/io/mosip/registration/clientmanager/service/MasterDataServiceImpl.java

```diff
@@ -51,6 +54,8 @@ public class MasterDataServiceImpl implements MasterDataService {
     public static final String REG_APP_ID = "REGISTRATION";
     public static final String KERNEL_APP_ID = "KERNEL";
 
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
let's create an instance of object mapper in AppModule and inject it via constructor wherever required.